### PR TITLE
[WIP] Add custom HTML to Course About page sidebar

### PR DESCRIPTION
**Pull request by stvstnfrd**
This allows course staff to add arbitrary HTML to the sidebar on the
About page.

We've been running this for a few years.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@caesar2164 This is totally untested, but it's a quick first pass at splitting out the feature.
Note: This is a merge against `ficus.master`.

File: lms/templates/courseware/course_about.html [hunk 1]
```diff
@@ -294,6 +294,7 @@ <h1>
             <li class="important-dates-item"><span class="icon fa fa-book" aria-hidden="true"></span><p class="important-dates-item-title">${_("Requirements")}</p><span class="important-dates-item-text prerequisites">${get_course_about_section(request, course, "prerequisites")}</span></li>
           % endif
         </ol>
+        <%static:optional_include_mako file="course_about_sidebar_extra.html" />
```
Comment (caesar2164) [inline lms/templates/courseware/course_about.html]: this is actually a different feature, and has already been separately merged upstream.

**Comment by stvstnfrd** [comment]
@caesar2164 Shall I close this now?

**Comment by caesar2164** [comment]
sure, I think I have it working pretty well on upstream PR.

**Pull request closed without merging**